model/clusters_mgmt/v1/azure_type.model

```diff
@@ -97,10 +97,13 @@ struct Azure {
 	// Immutable.
 	NetworkSecurityGroupResourceID String
 
+    // This is renamed to 'required_operators_authentication'.
+	// It brings concerns of; what if one of the operators stop being required anymore all the time?
+	// How do people see the design considering the future evolutions concerns?
 	// Defines how the operators of the cluster authenticate to Azure.
 	// Required during creation.
 	// Immutable.
-	OperatorsAuthentication AzureOperatorsAuthentication
+	RequiredOperatorsAuthentication AzureOperatorsAuthentication
```

Comment:
Even if this remains called OperatorsAuthentication, where you put the core operators information only, the same issue would happen

Reply:
+1

Reply:
The name would matter if you group all identities together, regardless of whether they are for core components or optional components.

Reply:
>  The name would matter if you group all identities together, regardless of whether they are for core components or optional components.

Yes in that case the name wouldn't matter, that's correct. In the master branch that's how the API currently looks like if you want to take a look. This is, all identities are grouped together and there's no differentiation between core and optional components. There is differentiation between control plane and data plane identities because they are conceptually different, configured in a different way and receive/return a different set of data